### PR TITLE
SCRUM-2000 Fix display of Last Loaded date for bulk load files

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

**Comment by oblodgett** [comment]
I don't think this solves the true fix we need... we need to have a separate date for the "last run time" and leave these other dates alone... lets look at this later.

**Comment by markquintontulloch** [comment]
@oblodgett - How about this?  Uses the last `loadFinished` date from the BulkLoadFileHistory for the "Last Loaded" date.  Avoids the need for creating and populating a new date field.

**Comment by oblodgett** [comment]
Ontology loads don't get the history object (maybe they should for this reason)

**Comment by markquintontulloch** [comment]
@oblodgett - 3rd time lucky?

File: src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoadFile.java [hunk 1]
```diff
@@ -21,6 +30,11 @@
 @AGRCurationSchemaVersion("1.2.4")
 public class BulkLoadFile extends GeneratedAuditedObject {
 	
+	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
+	@KeywordField(name = "dateLastLoaded_keyword", sortable = Sortable.YES, searchable = Searchable.YES, aggregable = Aggregable.YES, valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
+	@JsonView(View.FieldsOnly.class)
+	private OffsetDateTime dateLastLoaded;
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoadFile.java:36]: @markquintontulloch we need to add this to LinkML?

**Review by oblodgett — APPROVED** [review]

File: src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoadFile.java (hunk 1 above)
Comment (markquintontulloch) [reply src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoadFile.java:36]: yeah

**Pull request merged**